.gitignore

```diff
@@ -21,3 +21,4 @@ tmp
 *.swp
 .ruby-version
 .ruby-gemset
+.idea
```

Comment:
Non project-specific ignores do not belong in the `.gitignore`. You should pur them in your local ignore file.